.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I'm not a fan of committing vscode settings file, unless its to share environment settings to ease development for the team. But here it's for aesthetics only